### PR TITLE
Add multiclass classification test in raiwidgets

### PR DESCRIPTION
**Pull request by gaugup**
## Description

It seems we don't test multiclass scenario in `raiwidgets`. It might be good to have as we add more end points for widgets flask server.

## Checklist

- [ ] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

File: raiwidgets/tests/conftest.py [hunk 1]
```diff
@@ -73,3 +73,28 @@ def create_rai_insights_object_regression():
                   skip_cat_limit_checks=True)
     ri.compute()
     return ri
+
+
+@pytest.fixture(scope='session')
+def create_rai_insights_object_multiclass_classification():
```
Comment (imatiach-msft) [inline raiwidgets/tests/conftest.py:79]: nit: maybe shorten the name to:

create_rai_insights_multiclass_classification

I don't think object is needed in the name

File: raiwidgets/tests/test_responsibleai_dashboard_input.py [hunk 2]
```diff
@@ -220,3 +220,23 @@ def test_rai_dashboard_input_housing_causal_whatif_failure(
         assert "Failed to generate causal what-if," in \
             flask_server_prediction_output[
                 WidgetRequestResponseConstants.error]
+
+
+class TestResponsibleAIDashboardInputMultiClassClassification:
+    def test_rai_dashboard_input_iris_on_predict_success(
+            self, create_rai_insights_object_multiclass_classification):
+        ri = create_rai_insights_object_multiclass_classification
```
Comment (imatiach-msft) [inline raiwidgets/tests/test_responsibleai_dashboard_input.py:228]: tiny nit: maybe if we shorten the name the re-assign here won't be needed?
```suggestion
        rf = create_rai_insights_multiclass_classification.model
        test_data = create_rai_insights_multiclass_classification.test
```

File: raiwidgets/tests/test_responsibleai_dashboard_input.py [hunk 3]
```diff
@@ -220,3 +220,23 @@ def test_rai_dashboard_input_housing_causal_whatif_failure(
...
+        rf = ri.model
+        test_data = ri.test
```
Comment (imatiach-msft) [inline raiwidgets/tests/test_responsibleai_dashboard_input.py:230]: tiny nit: maybe if we shorten the name the re-assign here won't be needed?
```suggestion
        rf = create_rai_insights_multiclass_classification.model
        test_data = create_rai_insights_multiclass_classification.test
```

**Review by imatiach-msft — APPROVED** [review]

File: raiwidgets/tests/conftest.py (hunk 1 above)
Comment (gaugup) [reply raiwidgets/tests/conftest.py:79]: The fixture indeed returns an RAIInsights object. Not sure if there is need to shorten the fixture name unless it violates the linter rules.

Comment (gaugup) [reply raiwidgets/tests/conftest.py:79]: the other fixtures also carry the object. So keeping it the same,

File: raiwidgets/tests/test_responsibleai_dashboard_input.py (hunk 3 above)
Comment (gaugup) [reply raiwidgets/tests/test_responsibleai_dashboard_input.py:230]: closing this as responded to earlier comment,

**Pull request merged**